examples/v2/project_creation/project.py

```diff
@@ -74,6 +74,33 @@ def GenerateConfig(context):
               context.properties['concurrent_api_activation']
       }
   }, {
+      'name': 'get-iam-policy',
```

Comment:
Can you add a unit test that tests that the expected config blocks are generated ? I have created a test_project.py with unit tests for the features that I added to project.py. Also be sure to run it to make sure nothing existing was broken. We will need to set the jenkins automation to run these things automatically.